Doc/glossary.rst

```diff
@@ -939,11 +939,16 @@ Glossary
       modules, respectively.
 
    namespace package
-      A :pep:`420` :term:`package` which serves only as a container for
-      subpackages.  Namespace packages may have no physical representation,
+      A :term:`package` which serves only as a container for subpackages.
+      Namespace packages may have no physical representation,
       and specifically are not like a :term:`regular package` because they
       have no ``__init__.py`` file.
 
+      Namespace packages allow several individually installable packages to have a common parent package.
+      Otherwise, it is recommended to use a :term:`regular package`.
```

Comment:
I might suggest this change:
```suggestion
      If you don't have this need, it is recommended to use a :term:`regular package`.
```

But I’m also ok with the proposed wording.

Reply:
Ah, this goes back to wording that was omitted thanks to @willingc ’s review, so don’t apply my suggestion. We don’t want readers to ask themselves «do I have this need?!»